provider/source/data/debug/datetime/patterns/gregory/skeleton@1/ej/en.json

```diff
@@ -0,0 +1,19 @@
+{
+  "has_explicit_medium": true,
+  "variant_pattern_indices": [
+    3,
+    4,
+    4,
+    5,
+    6,
+    6
+  ],
+  "elements": [
+    "cccc, h a",
+    "ccc, h a",
```

Comment:
were these two patterns not generated before?

Reply:
no, only Ehm and Ehms were generated before.

Reply:
Can you explain where this change happens? It looks unrelated to me

Reply:
Since now h, hm, and hms patterns are shipped together in the same payload, I had to add Eh alongside Ehm and Ehms.